ci/macos/prepare.sh

```diff
@@ -7,4 +7,4 @@ set -x
 
 brew update
 brew upgrade cmake openssl@1.1
-brew install swig bison
+brew install swig bison flex
```

Comment:
https://docs.zeek.org/en/current/install.html#prerequisites should be updated to mention installing flex via brew.

Reply:
There are a number of docs issues outstanding, I'll collect them here:

- [ ] update prerequistes (CMake, Bison, Flex version, ...)
- [ ] add a minimal (trampoline?) section on Spicy to the Zeek docs which forwards to the Spicy docs

Reply:
Docs update is zeek/zeek-docs#117.